src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/casenotes/CaseNoteMapping.kt

```diff
@@ -10,10 +10,10 @@ import java.util.UUID
 data class CaseNoteMapping(
 
   @Id
-  val dpsCaseNoteId: UUID,
-
   val nomisCaseNoteId: Long,
 
+  val dpsCaseNoteId: UUID,
+
```

Comment:
We can now have multpile nomis case note ids mapped to the same dps id (when there has been a merge the nomis case notes are copied to the current booking but the DPS notes are not)